tasks/src/main/scala/dagr/tasks/ScatterGather.scala

```diff
@@ -71,36 +81,74 @@ object ScatterGather {
     /**
       * Produces a new [[Scatter]] from the existing scatter.  Partitions the the scattered objects of type Result
       * according to the given discriminator function.  The discriminator function maps an individual object of type
-      * Result to a key of type K.  The partitions are each a sequence of objects of type Result that share the same key
-      * of type K when the discriminator function is applied.
+      * Result to a key of type Key.  The partitions are each a sequence of objects of type Result that share the same key
+      * of type Key when the discriminator function is applied.
       *
       * The resulting Scatter[(Key, Seq[Result])] can be further mapped or gathered (or both!).
       */
     def groupBy[Key](f: Result => Key) : Scatter[(Key, Seq[Result])]
-  }
 
+    /**
+      * Converts this [[Scatter]] of scattered scatterers into a [[Scatter]] formed by the elements of the scatterers.
```

Comment:
This comment is trying far too hard to be cute.  Could just be:  "Flattens a `Scatter[Scatter[A]]` into a `Scatter[A]`."